### DISCUSSION TITLE
Disposition carry over?

### DISCUSSION THREAD

**Discussion by webfanatic** [DataKitchen/dataops-testgen#11] [Q&A]
I did a profiling run on a table group. I set the disposition (mute, etc) of some of the Anomalies. I then did another profiling run. When I look at the Anomalies from the second profiling run, should I expect to see the disposition carried over? The only other note, is that a new table was found during the second profiling run.

**Comment by cbloche** [comment]
If you had muted anything in the earlier run, you should not see that same anomaly appear in the later run.  Otherwise, you would not see your dispositions carry over.
